src/main/java/edu/harvard/iq/dataverse/confirmemail/ConfirmEmailServiceBean.java

```diff
@@ -102,11 +108,20 @@ private void sendLinkOnEmailChange(AuthenticatedUser aUser, String confirmationU
 
         try {
             String toAddress = aUser.getEmail();
-            /**
-             * @todo Move this to Bundle.properties.
-             */
-            String subject = BundleUtil.getStringFromBundle("notification.email.verifyEmail.subject");
-            mailService.sendSystemEmail(toAddress, subject, messageBody);
+            try {
+                Dataverse rootDataverse = dataverseService.findRootDataverse();
+                if (rootDataverse != null) {
+                    String rootDataverseName = rootDataverse.getName();
+                    // FIXME: consider refactoring this into MailServiceBean.sendNotificationEmail. Then we might need
```

Comment:
> might need what?

Sorry, the developer died while writing that comment. 😉 

I just finished off the comment in 5b75860. Thanks for noticing, @oscardssmith !